db/migrate/20220614120002_add_reset_pwd_attributes_to_users.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class AddResetPwdAttributesToUsers < ActiveRecord::Migration[7.0]
+  def change
+    change_table :users, bulk: true do |t|
+      t.string :reset_token_digest
+      t.datetime :reset_token_expires_at
```

Comment:
I would store `reset_sent_at` instead of `expires_at`

Reply:
I went with saving directly the datetime which after the token will be considered as expired rather than saving the sent time.
It felt more natural so the reset password logic simply fetches the expiration date from the database rather than inferring it.
Do you want me to change it?

Reply:
More details will be in #3535

Reply:
What happens if we want to change the time frame for when a token expires? We'd need to go back and update every entry in the db

Reply:
Hmm, haven't thought about that scenario, makes sense.
Good catch, I'll change the logic.